Cubical/Foundations/Univalence/Dependent.agda

```diff
@@ -0,0 +1,46 @@
+{-
+
+Dependent Version of Univalence
+
+The univalence corresponds to the dependent equivalences/isomorphisms,
+c.f. `Cubical.Foundations.Equiv.Dependent`.
+
+-}
+{-# OPTIONS --safe #-}
+module Cubical.Foundations.Univalence.Dependent where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Equiv
+open import Cubical.Foundations.Equiv.HalfAdjoint
+open import Cubical.Foundations.Equiv.Dependent
+open import Cubical.Foundations.Univalence
+
+private
+  variable
+    ℓ ℓ' : Level
+
+
+-- Dependent Univalence
+
+uaOver :
+  {A B : Type ℓ} {P : A → Type ℓ'} {Q : B → Type ℓ'}
+  (e : A ≃ B) (F : mapOver (e .fst) P Q)
+  (equiv : isEquivOver {P = P} {Q = Q} F)
+  → PathP (λ i → ua e i → Type ℓ') P Q
+uaOver {A = A} {P = P} {Q} e F equiv i x =
+  hcomp (λ j → λ
+    { (i = i0) → ua (_ , equiv x) (~ j)
+    ; (i = i1) → Q x })
+  (Q (unglue (i ∨ ~ i) x))
```

Comment:
A quick proof is `uaOver e F equiv = ua→ (λ a → ua (_ , equiv a))`. But your definition gives a smaller term overall, so maybe it's better to leave it as is.

Reply:
I'll leave it as a comment.